package.json

```
@@ -19,7 +19,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm test"
+      "pre-push": "npm test",
+      "precommit": "npm run docs"
```

Could we do it on `pre-push` instead?

```suggestion
    "pre-push": "npm run docs && npm test"
```